meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -151,6 +151,10 @@ OSTREE_KERNEL_ARGS:mx6ull-generic-bsp ?= "console=tty1 console=ttymxc0,115200 ${
 OSTREE_KERNEL_ARGS:mx7d-generic-bsp ?= "console=tty1 console=ttymxc0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
 OSTREE_KERNEL_ARGS:mx7ulp-generic-bsp ?= "console=tty1 console=ttyLP0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
 
+PREFERRED_VERSION_meson ?= "0.61.3"
```

Comment:
I think we don't need to set the default version and we should pick the one available in oe-core.
With this we will need to continue update the version to keep track the one in oe-core.